### PR TITLE
#162985659 latest meetups 

### PR DESCRIPTION
**Pull request by Mupa1**
__What does this PR do?__

Adds latest meetups to landing page

__Description of task to be completed__
Have the following goals achieved:
* A responsive landing page
* Add latest meetups
* Add a search bar
* Create a link to all meetups page

__How should this be manually tested?__
1. Cloning the repo ```https://github.com/Mupa1/Questioner.git``` 
2. Go to the UI folder and open index.html on a browser

__Relevant pivotal tracker stories?__
#162985659

__Screenshots__
landing_page

### CODE REVIEW

**Comment by Mfalm3** [comment]
Awesome UI @Mupa1

**Comment by wachiranduati** [comment]
That is really nice :+1:

File: UI/css/style.css [hunk 1]
```diff
@@ -56,29 +73,102 @@ header li{
...
+    #searchbar form input[type="text"]{
+        width: 100%;
+        margin-bottom: 10px;
+    }
+    #searchbar button{
+        display: block;
+        width: 100%
+    }
+
 }
```
Comment (Ogollah) [inline UI/css/style.css]: new line here.

**Comment by Ogollah** [comment]
Kindly post screenshot for a mobile version.

**Comment by SolomonMacharia** [comment]
Really nice home page.

**Comment by Mupa1** [comment]
@Ogollah sorry, please explain, i did not understand what you meant by new line, please expound

Here are the screenshots for mobile view

screenshot from 2019-01-05 14-58-36

screenshot from 2019-01-05 14-53-54

screenshot from 2019-01-05 14-57-40

File: UI/css/style.css (hunk 1 above)
Comment (Mupa1) [reply UI/css/style.css]: Hi @Ogollah, What about the new line?

**Comment by Mupa1** [comment]
@Mfalm3 @wachiranduati @SolomonMacharia thank guys

**Comment by Ogollah** [comment]
after the last code, you need a next line. which is blank with no code.

**Pull request merged**